algos.yaml

```diff
@@ -352,6 +438,28 @@ bigann-1B:
                 "nprobe=256,quantizer_efSearch=64",
                 "nprobe=256,quantizer_efSearch=128",
                 "nprobe=256,quantizer_efSearch=512"] 
+    buddy-t1:
+      docker-tag: billion-scale-benchmark-faissconda
```

Comment:
Did you intend to use faissconda docker image? Should I ignore the Docker.pqbuddy file?